src/js/GenericPointer.js

```diff
@@ -0,0 +1,32 @@
+/******************************************************************************
+ * Aladin Lite project
+ * 
+ * File GenericPointer.js
+ * 
+ ******************************************************************************/
+
+import { SimbadPointer } from "./SimbadPointer.js";
+import { PlanetaryFeaturesPointer } from "./PlanetaryFeaturesPointer.js";
+
+// allow to call either Simbad or Planetary features Pointers
+export let GenericPointer = (function (view, e) {
+    const xymouse = view.imageCanvas.relMouseCoords(e);
+    let radec = view.wasm.screenToWorld(xymouse.x, xymouse.y);
+    if (radec) {
+        // sky case
+        if (view.aladin.getBaseImageLayer().properties.isPlanetaryBody === false) {
```

Comment:
Maybe we should provide a new isPlanetaryBody() method for the ImageSurvey class and call `view.aladin.getBaseImageLayer().isPlanetaryBody()` instead.

Appart from that modification (so maybe for another PR),
I notice that this generic pointer will fail for an aladin lite instance that only contain a fits file (i.e. a ImageFITS object as the base image layer, not an ImageSurvey object). We should maybe implement isPlanetaryBody() for imageFITS as well, but how can be know a fits file refer to a planet ? @tboch

Reply:
We could ask the user if there is no BaseImageLayer ? They surely know

Reply:
Something like "We detected no base layer. Do you want to do a search on planetary or sky databases?" and then 
![image](https://user-images.githubusercontent.com/16650466/236402264-14cf92b4-10e5-4918-80d3-73bca27347db.png)

Reply:
I was referring that view.aladin.getBaseImageLayer() can either provide a ImageSurvey or an ImageFITS object depending on what is stored as the base layer (a HiPS for the first case, a fits file for the second case). Here we are suspecting  getBaseImageLayer will return an ImageSurvey object (because an ImageFITS has not a properties field, only ImageSurveys have). So this code will throw an error if the base layer is a FITS file (something like properties is undefined).
We can think of having a more generic method isPlanetaryBody() on the ImageSurvey class and later on the ImageFITS class as well (I suspect there should be FITS header keywords that tell if the fits refers to a planet, and if not found, we could consider the default case as: refers to the sky).

Reply:
Oh ok I get it now

Reply:
Do we open an issue so that we don't forget when this PR will be closed ?

Reply:
And also you are right, what should we do if there is just no layers ?
Normally we put the DSS HiPS as the base if the user tell us nothing (or all the layers he provides cannot be loaded because of various errors). I did that but this 'default' behaviour can still be discussed, maybe we want to have no layer by default. I guess this was the default behaviour of aladin lite v2 as well (to put the DSS color even if the user ask for nothing), that is why it is the same in the v3.